packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -25,6 +25,14 @@ export const prodParasKusama: EndpointOption[] = [
       OnFinality: 'wss://altair.api.onfinality.io/public-ws'
     }
   },
+  {
+    info: 'amplitude',
+    homepage: 'https://pendulumchain.org/amplitude',
+    paraId: 2124,
+    text: 'Amplitude',
+    isUnreachable: true,
+    providers: {} // Working on making this live ASAP
+  },
```

Comment:
This does look like it is there -

https://github.com/polkadot-js/apps/blob/a6cc58e582ea60c3f220e23b36d563220b1ca136/packages/apps-config/src/endpoints/productionRelayKusama.ts#L28-L35